site/2017/10/24/odm-1-2-0-and-1-1-7.rst

```diff
@@ -0,0 +1,81 @@
+Doctrine MongoDB ODM 1.2.0 and 1.1.7 Released
+=============================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.2.0 <https://github.com/doctrine/mongodb-odm/releases/tag/1.2.0>`_ and
+`1.1.7 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.7>`_.
+
+MongoDB ODM 1.1.7
+-----------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/1.1.x/CHANGELOG-1.1.md#117-2017-10-23>`_.
+A full list of issues and pull requests included in this release may be found in the
+`1.1.7 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.1.7>`_.
```

Comment:
Do you typically link to an issue query, or did you mean to use https://github.com/doctrine/mongodb-odm/milestone/26?closed=1 (I assume the last open issue will be closed soon)?

Reply:
I used the link we had in the previous blog post as well as the changelogs simply because it's easier to adjust for subsequent releases (milestone name vs. id).